docs/sections/contributor_guide/code_quality.rst

```diff
@@ -1,5 +1,36 @@
-************
 Code Quality
-************
+============
```

Comment:
Built this page from content moved from the _Developer Setup_ page, with some additions.